temporalcli/commandsgen/commands.yml

```diff
@@ -394,6 +394,160 @@ commands:
+          --deployment-series-name YourDeploymentSeriesName \
+          --deployment-build-id YourDeploymentBuildId \
+          --report-reachability
+      ```
+    option-sets:
+      - deployment-reference
+    options:
+      - name: report-reachability
+        type: bool
+        description: |
+          Flag to include reachability information of a Deployment.
+
+  - name: temporal deployment get-current
+    summary: Show the current Deployment
+    description: |
+      +---------------------------------------------------------------------+
+      | CAUTION: Worker Deployment is experimental. Deployment commands are |
+      | subject to change.                                                  |
+      +---------------------------------------------------------------------+
+
+      Gets the current Deployment for a Deployment Series Name. Whether a
+      Deployment is current or not can affect which Workers will execute
+      Tasks of an existing or new Workflow.
+
+
+      ```
+      temporal deployment get-current [options]
+      ```
+
+      For example:
+
+      ```
+      temporal deployment get-current \
+          --deployment-series-name YourDeploymentSeriesName
+      ```
+    options:
+      - name: deployment-series-name
+        type: string
+        description: Series Name for the current Deployment.
+        required: true
+
+  - name: temporal deployment list
+    summary: Enumerate Deployments in the client's namespace
+    description: |
+      +---------------------------------------------------------------------+
+      | CAUTION: Worker Deployment is experimental. Deployment commands are |
+      | subject to change.                                                  |
+      +---------------------------------------------------------------------+
+
+      List existing Deployments in the client's namespace, optionally
+      filtering them by Deployment Series Name.
+
+
+      ```
+      temporal deployment list [options]
+      ```
+
+      For example, adding an optional filter:
+
+      ```
+      temporal deployment list \
+          --deployment-series-name YourDeploymentSeriesName
+      ```
+    options:
+      - name: deployment-series-name
+        type: string
+        description: Series Name to filter Deployments.
+
+  - name: temporal deployment update-current
+    summary: Change the current Deployment
+    description: |
+      +---------------------------------------------------------------------+
+      | CAUTION: Worker deployment is experimental. Deployment commands are |
+      | subject to change.                                                  |
+      +---------------------------------------------------------------------+
+
+      Sets the current Deployment for a given Deployment Series. Whether a
+      Deployment is current or not can affect which Workers will execute
+      Tasks of an existing or new Workflow.
+
+      ```
+      temporal deployment update-current [options]
```

Comment:
Same as above, `set-current` seems more straightforward to me and matches the API